src/api/PlainUpload.js

```diff
@@ -78,7 +65,76 @@ class PlainUpload extends BaseUpload {
             return;
         }
 
-        this.baseUploadErrorHandler(error, this.makePreflightRequest.bind(this));
+        const fileName = this.file ? this.file.name : '';
+
+        // TODO(tonyjin): Normalize error object and clean up error handling
+        let errorData = error;
```

Comment:
this hack needs to be cleaned.
This function is called both from an axios error response and also from native XHR error response.